code/style.css

```diff
@@ -36,50 +74,114 @@ p {
 input {
   box-sizing: border-box;
   border: none;
-  border-radius: 4px 0 0 4px;
-  background: #e5e9ff;
-  color: #0026ff;
-  padding: 16px;
-  font-size: 16px;
-  font-family: 'Montserrat';
+  border-radius: 14px;
+  color: #303030;
+  padding: 18px;
+  font-size: 18px;
+  font-family: "skolar-sans-latin-condensed", sans-serif;
   font-weight: 600;
+  font-style: normal;
   line-height: 26px;
   flex: 1;
+  text-transform: uppercases;
+    background: linear-gradient(-45deg, #edd7d0, #f6dce5, #d8e8ed, #e7f4f1);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+  height: 8S0vh;
+}
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+
+input:focus,
+select:focus {
+  outline: none;
+}
+
+select {
+  border: none;
+  border-radius: 14px;
+  color: #303030;
+  font-size: 20px;
+  font-weight: 600;
+  height: 60px;
+  font-family: "skolar-sans-latin-condensed", sans-serif;
+  font-weight: 600;
+  margin-bottom: -6px;
+  font-style: normal;
+  width: 100%;
+  text-align: center;
+  padding: 0 20px;
+  cursor: pointer;
+  text-transform: uppercases;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: linear-gradient(-45deg, #edd7d0, #f6dce5, #d8e8ed, #e7f4f1);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+  height: 8S0vh;
+}
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
```

Comment:
The fact that all the elements have the same gradient as the background is a really great touch.